logos/README.md

```diff
@@ -41,16 +41,50 @@ poetry env activate
 poetry install
 ```
 
+### Configuration
+
+Create the file 'logos.local.yml'. This file will contain the users
+and their respective models. You can have a look into 'logos.example.yml'
+as a reference. Basically, for a single user with one model
+this looks like this:
+
+```yaml
+users:
+  - name: "Name or Mail (unique)"
+    keys:
+      - model: "Model Name"
+        key: "API-Key"
+        pwd: "Password for this model (can be empty if private is false)"
+        provider: "Where this LLM runs (default is openai)"
+        quota: "How many prompts may be processed by this model (default=-1, all)"
+        private: true or false, depending if this model should not be accessible 
+                    without further user authentication
+```
+
+The password stored has to be hashed with bcrypt:
+```py
+import bcrypt
+bcrypt.hashpw(password, bcrypt.gensalt())
+```
+
+If you don't want to provide a user, use the name "default". With no user provided,
+logos will the automatically look under this branch.
+
 ## Running the Service
 
+### Additional Logos API Parameters
+Provide the user with the key "user" and the password with the key "password" 
+in the payload of your request to access user LLMs with or without password. If you don't provide
+the user, logos will look in the "default" branch of the config.
+
 ### Development
 
 ```bash
-poetry run fastapi dev
+poetry run fastapi dev "./src/logos/main.py"
 ```
 
 ### Production
 
 ```bash
-poetry run fastapi run
+poetry run fastapi run "./src/logos/main.py"
```

Comment:
I think the file path could be removed here, if changes to pyproject.toml are made accordingly.